scripts/convert_lit_checkpoint.py

```diff
@@ -196,6 +196,20 @@ def maybe_unwrap_state_dict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, t
     return lit_weights.get("model", lit_weights)
 
 
+def maybe_raise_finetune_warning(lit_weights: Dict[str, torch.Tensor], checkpoint_dir: Path) -> None:
+    weight_names = set([wk.split(".")[-1] for wk in lit_weights.keys()])
+    # adapter
+    if "gating_factor" in weight_names:
+        raise Exception(f"Converting models finetuned with adapter not yet support")
+    # adapter v2
+    elif "adapter_bias" in weight_names:
+        raise Exception(f"Converting models finetuned with adapter_v2 not yet support")
```

Comment:
```suggestion
        raise NotImplementedError(f"Converting models finetuned with adapter v2 is not supported")
```

Reply:
implemented in [d16241b](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/d16241bda4a44da3135a3f63d7b3c9f116a08494)